src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainer.groovy

```diff
@@ -374,7 +374,7 @@ class DockerCreateContainer extends AbstractDockerRemoteApiTask {
         }
 
         if(getLabels()) {
-            containerCommand.withLabels(getLabels())
+            containerCommand.withLabels(getLabels().collectEntries { [it.key, it.value.toString()] })
```

Comment:
Should we do a `toString()` on the key as well?

Reply:
I'll check this.